### ISSUE TITLE
Docs update 

### ISSUE THREAD

**Issue by bologer** [rosell-dk/webp-convert#97]
By [official Google docs](https://developers.google.com/speed/webp/docs/precompiled), it seems that library can convert the following: JPEG, PNG or TIFF, but in the documentation (or at least the README) it says only first two.

**Comment by rosell-dk** [comment]
WebP Convert checks the file extension and only allow files with png, jpg or jpeg extension to be converted. So docs are alright... But WebP Convert could be changed so it does not filter out TIFF.

**Comment by bologer** [comment]
Makes sense, it seems like you are keeping allowed extensions in the helper. 
What about moving it in some abstract class and each converter, e.g. WebP extending this class. It would follow polymorphism and we have a choice to define allowed extension list per converter (as it is usually like that). 

Such as: 

```php
class WebP extends BaseConverter {
    $allowedExtensions = ['image/jpeg', 'image/jpg', 'image/png', 'image/tiff'];
...
} 
```


```php
class BaseConverter {
    protected $allowedExtensions = [];
}

**Comment by rosell-dk** [comment]
Yes, I'm actually on it (the BaseConverter). The 2.0.0 branch uses that pattern. I'm not completely through with that refactoring

**Comment by rosell-dk** [comment]
Just released 2.0.0-beta. I will probably release 2.0.0 tomorrow
